mysql/my.cnf

```diff
@@ -0,0 +1,3 @@
+[mysqld]
+general_log = 1
+general_log_file = /var/lib/mysql/general.log
```

Comment:
Do the defaults suffice for this? Seeing a config for MySQL for just log files inside a container doesn't seem worth it.